server.js

```diff
@@ -29,24 +27,18 @@ phoenix.userLogin(
  */
 app = express();
 
-const appConfig = require('./config')();
-const router = require('./api/router');
+const appConfig = require('./config')(); 
 const smsConfigsLoader = require('./config/smsConfigsLoader');
 
-app.locals.northstarClient = new NorthstarClient({
-  baseURI: process.env.DS_NORTHSTAR_API_BASEURI,
-  apiKey: process.env.DS_NORTHSTAR_API_KEY,
-});
+const router = require('./api/router');
 
-app.locals.phoenixClient = new PhoenixClient({
-  baseURI: process.env.DS_PHOENIX_API_BASEURI,
-  username: process.env.DS_PHOENIX_API_USERNAME,
-  password: process.env.DS_PHOENIX_API_PASSWORD,
-});
+const CampaignBotController = rootRequire('api/controllers/CampaignBotController');
 
 // Retrieves all SMS config files before starting server.
 smsConfigsLoader(() => {
   const port = (process.env.PORT || 5000);
+  // @todo We'll need to loop through all campaignBots and store as array.
+  app.locals.campaignBotController = new CampaignBotController(41);
```

Comment:
Ya -- this needs to turn into an environment variable. it's the default CampaignBot ID (from the Gambit Jr API) that stores the copy to use for each response message - e.g. `this.bot.msg_ask_quantity`, `this.bot_msg_menu_completed`